ganttproject/src/biz/ganttproject/impex/csv/XlsReaderImpl.java

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2017 Alexandr Kurutin, BarD Software s.r.o
+
+This file is part of GanttProject, an opensource project management tool.
+
+GanttProject is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+GanttProject is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
+*/
+package biz.ganttproject.impex.csv;
+
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Workbook;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+
+/**
+ * @author torkhov
+ */
+class XlsReaderImpl implements SpreadsheetReader {
+
+  private final Workbook myBook;
+  private final List<String> myColumnHeaders;
```

Comment:
The only use of this var is in  `initializeHeader()` where it can be just directly passed to. Remove?